lcov_parser.rb

```diff
@@ -45,11 +45,14 @@ def self.parse(filepath)
                         executableLines: 0,
                         lineCoverage: 0,
                         functions: [] }
-        target[:name] = Regexp.last_match(1).strip.empty? ? "No Name" : Regexp.last_match(1).strip
+        tn_value = Regexp.last_match(1).strip
+        target[:name] = tn_value.empty? ? "No Name" : tn_value
 
       when /^SF:(.+)/ # Source file
         current_filename = $LAST_MATCH_INFO[1].gsub(%r{^\./}, "")
+        file_name_only = File.basename(current_filename)
```

Comment:
At first glance I couldn't understand what the file_name_only name does.
IMO it's more appropriate as below, don't you think?
- `file_basename`

basename fit for this purpose:
- https://en.wikipedia.org/wiki/Basename